src/ast/document-descriptor.ts

```diff
@@ -118,7 +118,7 @@ export class Document implements Feature {
     for (const scannedFeature of base.entities) {
       i += 1;
       if (scannedFeature instanceof ScannedImport) {
-        const scannedDoc = dependenciesByUrl.get(scannedFeature.url)!;
```

Comment:
I've never seen this before, is this valid TypeScript syntax? what is this trailing exclamation mark doing?